test/topology/test_gossiper.py

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright (C) 2023-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+
+from test.pylib.manager_client import ManagerClient
+import pytest
+
+@pytest.mark.asyncio
+async def test_gossiper_live_endpoints(manager: ManagerClient) -> None:
```

Comment:
Let's merge these test cases into one, I see the flow (which servers we stop etc.) is pretty much identical, just the checks are different. We can do the checks together in a single test case.